cocos/base/CCDirector.cpp

```diff
@@ -260,7 +254,7 @@ void Director::setGLDefaultValues()
     setProjection(_projection);
 
     // set other opengl default values
-    glClearColor(_clearColor.r, _clearColor.g, _clearColor.b, _clearColor.a);
+    setClearColor(Color4F::BLACK);
```

Comment:
If not cache value of clear color, then setGLDefaultValues() will override it.
And setGLDefaultValues() will be invoked every frame.